docs/dev/zksync-v2/blocks-and-time.md

```diff
@@ -1,3 +1,24 @@
 # Block numbers and time
 
-TODO.
+On ZkSync there are two types of blocks: L2 blocks and L1 batches.
+
+## L2 blocks
+
+L2 block is similar to a common block on L1. L2 block time is almost constant, like a few seconds.
+In this documentation by default the word "block" actually means "L2 block" including API.
```

Comment:
"word ... actually means" makes it sound like we wrote the documents incorrectly and are unable to use the correct term outselves.
To make it sound better we need to work on phrasing, i.e. "For simplicity, when we mention "block" in this documentation, "L2 block" is always assumed, unless specified otherwise".
"including API" part is a bit confusing to me -- what's so special about API?